tests/unit/helpers.js

```diff
@@ -43,6 +43,8 @@ export function makeAuthInfo(accessKey) {
             + 'cd47ef2be',
         accessKey2: '79a59df900b949e55d96a1e698fbacedfd6e09d98eacf8f8d5218e7'
             + 'cd47ef2bf',
+        accessKey3: '79a59df900b949e55d96a1e698fbacedfd6e09d98eacf8f8d5218e7'
```

Comment:
This was added to make sure tests in [tests/unit/api/objectDelete](https://github.com/scality/S3/blob/master/tests/unit/api/objectDelete.js) pass with the new checks on the IDs. The objectDelete tests use the makeAuthInfo helper function to generate canonical IDs, including makeAuthInfo('accessKey3'), but accessKey3 did not previously map to a valid canonical ID.